dascore/core/coords.py

```
@@ -2810,15 +2806,15 @@ def _max(self):
 
 def get_coord(
     *,
-    data: ArrayLike | None | np.ndarray | BaseCoord = None,
-    values: ArrayLike | None | np.ndarray = None,
+    data: ArrayLike | np.ndarray | BaseCoord | None = None,
+    values: ArrayLike | np.ndarray | None = None,
     start=None,
     min=None,
```

Ruff (0.16.1)

[error] Function argument `min` is shadowing a Python builtin

(A002)